gemrb/core/Scriptable/Actor.cpp

```diff
@@ -3706,19 +3706,19 @@ void Actor::RollSaves()
 }
 
 //saving throws:
-//type      bits in file    order in stats
-//0  spells            1    4
-//1  breath            2    3
-//2  death             4    0
-//3  wands             8    1
-//4  polymorph        16    2
+//type	  bits in file	order in stats
+//0  spells			1	4
+//1  breath			2	3
+//2  death			 4	0
+//3  wands			 8	1
+//4  polymorph		16	2
 
 //iwd2 (luckily they use the same bits as it would be with bg2):
 //0 not used
 //1 not used
-//2 fortitude          4   0
-//3 reflex             8   1
-//4 will              16   2
```

Comment:
The search&replace you did had some fallout, of which this double chunk is the only one needing a revert.